tools/pipelines/templates/build-npm-package.yml

```
@@ -133,6 +137,9 @@ stages:
                 Test=${{ parameters.taskTest }}
                 BuildDoc=${{ parameters.taskBuildDocs }}
                 TestCoverage=$(testCoverage)
+              
+              Publish Parameters:
+                nonScopedPackages=${{join(', ',parameters.nonScopedPackages)}}
```

```suggestion
                nonScopedPackages=${{ join(', ', parameters.nonScopedPackages) }}
```
nit: consistent formatting